config/fedora.yml

```diff
@@ -1,22 +1,22 @@
 development:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8984') %>/rest
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8984/fcrepo/rest') %>
   base_path: /dev
 test:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080') %>/rest
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080/fcrepo/rest') %>
   base_path: /test
 production:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080') %>/rest
-  base_path: /prod
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080/fcrepo/rest') %>
   request: { timeout: 1000, open_timeout: 1000 }
+  base_path: /prod
 staging:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080') %>/rest
-  base_path: /staging
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_FEDORA_URL', 'http://localhost:8080/fcrepo/rest/staging') %>
```

Comment:
Is `/fcrepo/rest/staging` right since we have `base_path: /staging` later? Not that I think it matters for staging, since it's pretty much always going to be caught through environment

Reply:
I unraveled a lot of the stuff that appends tokens to the path. I removed base_path and made everything explicit

Reply:
I did fix the production entry to match the prod core URL